libraries/AC_WPNav/AC_Circle.cpp

```diff
@@ -23,6 +23,13 @@ const AP_Param::GroupInfo AC_Circle::var_info[] = {
     // @User: Standard
     AP_GROUPINFO("RATE",    1, AC_Circle, _rate,    AC_CIRCLE_RATE_DEFAULT),
 
+    // @Param: POINTLOOP
+    // @DisplayName: Stopping point loop enable
+    // @Description: Set the center of the circle to the stopping point.
+    // @Values: 0:Disabled,1:Enabled
+    // @User: Standard
+    AP_GROUPINFO_FLAGS("POINTLOOP",    2, AC_Circle, _stopping_point_center_enable, 0, AP_PARAM_FLAG_ENABLE),
```

Comment:
Flag is wrong, this parameter isn't an enabler of this library.

Reply:
yup, flag needs to be removed.

Reply:
Is it to be AP_GROUPINFO?

Reply:
yes. This is just a regular param.